connectors/src/connectors/slack/bot.ts

```diff
@@ -42,8 +42,9 @@ export async function botAnswerMessageWithErrorHandling(
   slackChannel: string,
   slackUserId: string,
   slackMessageTs: string,
-  slackThreadTs: string | null
-): Promise<Result<AgentGenerationSuccessEvent, Error>> {
+  slackThreadTs: string | null,
```

Comment:
NIT: small refactor to have an object over a list of params for this function that gets now 7 params ? 🙏🏻